assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -55,7 +55,9 @@ spec:
             - "--enable-auto-update=false"
             - "--enable-default-cluster-version=true"
             - "--kubeconfig=/etc/openshift/kubeconfig/kubeconfig"
-            - "--listen="
+            - "--listen=0.0.0.0:9099"
```

Comment:
Since we are running within the same pod, we could simply listen on `127.0.0.1` instead of all interfaces.  Doing so is more secure.

Reply:
Updated to use localhost.